packages/blade/src/components/Typography/Heading/Heading.tsx

```diff
@@ -17,6 +17,7 @@ type HeadingCommonProps = {
   type?: TextTypes;
   contrast?: ColorContrastTypes;
   children: StringChildrenType;
+  textAlign?: BaseTextProps['textAlign'];
```

Comment:
Should we avoid adding on `Heading` right now? I was thinking if we can remove from `Text` as well and just rely on parent's textAlign 🤔 

cause textAlign will stop working when we support something like `as="span"` no?

Reply:
> Should we avoid adding on Heading right now? I was thinking if we can remove it from Text as well and just rely on parent's textAlign 🤔

Should be fine no harm on adding textAlign. 

> cause textAlign will stop working when we support something like as="span"
Span seems to respect textAlign property, it doesn't ignore it.

Reply:
https://codesandbox.io/s/razorpay-blade-basic-forked-kmoo5u?file=/src/App.tsx:386-440

It will ignore no? or did you mean when we render Text component as span it works differently?

Reply:
Oh i see, I did the inheritance : 

```
      <div style={{ textAlign: "center" }}> 
        <span>hi</span>
      </div>
```

But anyway, it's fine it it ignores too if thats the default behaviour of CSS.